### DISCUSSION TITLE
How to suppress title text?

### DISCUSSION THREAD

**Discussion by mgutz** [herbstluftwm/herbstluftwm#1496] [Q&A] [answered]
I want to show title decoration on max layout, but not text. I tried `attr theme.title_font 'mono:pixelsize=0'` and that results in a big font. Setting `pixelsize=1` almost hides it. Using same fg and bg color doesn't work when using transparency.

I'm on latest git.

**Comment by t-wissmann** [comment, marked as answer]
I think it's not really possible at the moment. It will be possible as soon as CSS-style theming (#1457) is supported. It might be hackable now with an unpleasent workaround:
```
hc attr theme.title_depth -40 ; hc attr theme.title_height 50
```
this shows tabs/decoration with a height of 10 pixels (50 - 40), but draws the window titles beneath the windows themselves, so it is hidden.
